Tests/FunctionalTests/fss-signin-page.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { mkdir, mkdirSync } from 'fs';
+import { chromium, Browser,BrowserContext, Page } from 'playwright'
+const { autoTestConfig } = require('./appSetting');
+const {pageObjectsConfig} = require('./pageObjects');
+import {tmpdir} from 'os'
+
+describe('Test Sign In Page Scenario', () => {
+    jest.setTimeout(120000);
+    let browser: Browser;
+    let context: BrowserContext;
+    let page: Page;
+
+    beforeEach(async () => {
+        browser = await chromium.launch();
+          context = await browser.newContext();
+          page = await context.newPage();         
+          await page.goto(autoTestConfig.url)
+      })
+
+      afterEach(async () => {
+        await page.close()
+        await context.close()
+        await browser.close()
+    })
+
+    //Function to sign in portal
+    //==================START==============================
+    async function LoginPortal(username: string, password: string)
+    {
+      
+      const [popup] = await Promise.all([
+        page.waitForEvent('popup')
+        ]);
+      
+      popup.setDefaultTimeout(60000);
+      popup.setViewportSize({'width' : 800,'height' : 1024})
+      await popup.waitForSelector(pageObjectsConfig.loginPopupSignInEmailSelector)        
+      popup.fill(pageObjectsConfig.loginPopupSignInEmailSelector, username)
+      await popup.waitForSelector(pageObjectsConfig.loginPopupNextButtonSelector)      
+      popup.click(pageObjectsConfig.loginPopupNextButtonSelector)
+      await popup.waitForSelector(pageObjectsConfig.loginPopupSignInPasswordSelector)
+      popup.fill(pageObjectsConfig.loginPopupSignInPasswordSelector, password)
+      await popup.waitForSelector(pageObjectsConfig.loginPopupSignInButtonSelector)
+      popup.click(pageObjectsConfig.loginPopupSignInButtonSelector)     
+      
+    }  
+    //===============END===================================
+
+    it('User clicks Sign in link with valid credentails should display FirstName after login successfully', async () => {
+    
+        page.click(pageObjectsConfig.loginSignInLinkSelector);
+        try {
+          await LoginPortal(autoTestConfig.user, autoTestConfig.password);
+        }
+        catch (e) {
+          let temppath = tmpdir();
+          
+          await page.screenshot({
+            path: temppath + "\failedtest.jpeg"
```

Comment:
Would it be more useful to include the test name in the failed test JPG ?

Reply:
Added Test name for the failed test. Thanks